guide/14-deep-learning/doccano setup guide.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>Suggestion:</p><p> </p><blockquote>to reduce redundant information &amp; improving user experience.</blockquote><p> </p><p>This notebook is mostly about using doccano rather than deploying it,</p><p>as that is an automated process, with minor instructions, which are already given in 'readme.md'.</p><p>so a more intuitive name of the notebook, for a user can be like "Getting started with doccano.ipynb". </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>